config/demo-playlist.json

```diff
@@ -99,6 +99,23 @@
         }
       }
     },
+    {
```

Comment:
It's not necessary for this PR. I can remove it if you'd like the PR to be more pure, and send it along as its own PR instead...